src/less/box-shadow.less

```diff
@@ -4,30 +4,42 @@
 // Opinionated defaults for box shadows
 
 @boxShadowDefaultXOffset: 0;
-@boxShadowDefaultYOffset: @size_4xs;
 
-@boxShadowLightBlack: fade(@neutral_black, 40%);
-@boxShadowMediumBlack: fade(@neutral_black, 50%);
-@boxShadowHeavyBlack: fade(@neutral_black, 70%);
+@boxShadowYOffsetS: @size_4xs;
+@boxShadowYOffsetM: 0.1875rem;
+@boxShadowYOffsetL: 0.3125rem;
 
-@boxShadowBlurRadiusS: @size_xs;
-@boxShadowBlurRadiusM: @size_s;
-@boxShadowBlurRadiusL: @size_m;
+@boxShadowBlurRadiusS: 0.1875rem;
+@boxShadowBlurRadiusM: 0.375rem;
+@boxShadowBlurRadiusL: @size_s;
```

Comment:
These three values are kind of confusing together. I see how the value of `S` and `M` relate. But, in order for me to see how `M` relates to `L` I have to go hunt down the value of `@size_s`.

Reply:
Great call - updated